docs/kubernetes/beginners-guide-to-kubernetes-part-4-controllers/index.md

```diff
@@ -53,62 +53,61 @@ spec:
+To see for yourself that the images have updated, you can grab the Pod name from the `get pods` list:
 
-    NAME                DESIRED   CURRENT   READY   AGE
-    apache-replicaset   5         5         5       86s
+    kubectl get pods
 
-You can view the Pods the ReplicaSet created by issuing the `get pods` command:
+    NAME                                 READY   STATUS    RESTARTS   AGE
+    apache-deployment-574c8c4874-8zwgl   1/1     Running   0          8m36s
+    apache-deployment-574c8c4874-9pr5j   1/1     Running   0          8m36s
+    apache-deployment-574c8c4874-fbs46   1/1     Running   0          8m34s
+    apache-deployment-574c8c4874-nn7dl   1/1     Running   0          8m36s
+    apache-deployment-574c8c4874-pndgp   1/1     Running   0          8m33s
 
-    kubectl get pods
+Issue the `describe` command to view all of the available details of the Pod:
 
-You should see output like the following:
+    kubectl describe pod apache-deployment-574c8c4874-pndgp
 
-    NAME                      READY   STATUS    RESTARTS   AGE
-    apache-replicaset-5rsx2   1/1     Running   0          31s
-    apache-replicaset-8n52c   1/1     Running   0          31s
-    apache-replicaset-jcgn8   1/1     Running   0          31s
-    apache-replicaset-sj422   1/1     Running   0          31s
-    apache-replicaset-z8g76   1/1     Running   0          31s
+You'll see a long list of details, of which the container image is included:
 
-To delete a ReplicaSet, issue the `delete replicaset` command:
+    ....
 
-    kubectl delete replicaset apache-replicaset
+    Containers:
+      apache-container:
+        Container ID:   docker://d7a65e7993ab5bae284f07f59c3ed422222100833b2769ff8ee14f9f384b7b94
+        Image:          httpd:2.4.38
 
-If you issue the `get pods` command, you will see that the Pods the ReplicaSet created are in the process of terminating:
+    ....
 
-    NAME                      READY   STATUS        RESTARTS   AGE
-    apache-replicaset-bm2pn   0/1     Terminating   0          3m54s
 
-In the above example, four of the Pods have already terminated, and one is in the process of terminating.
+For more information on Deployments, visit the [Kubernetes Deployments API documentation](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#deployment-v1-apps)
 
-For more information on ReplicaSets, view the [Kubernetes ReplicaSets API documentation](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#replicaset-v1-apps).
+## ReplicaSets
 
-## Deployments
+{{< note >}}
+Kubernetes now [recommends](https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/#when-to-use-a-replicaset) the use of Deployments instead of ReplicaSets. Deployments provide declarative updates to Pods among other features that allow you to define your application in the spec section. In this way, ReplicaSets have essentially become deprecated.
```

Comment:
Comma after Pods, and comma after features.

Reply:
updated